src/main/java/org/commonjava/util/partyline/JoinableFileManager.java

```diff
@@ -203,24 +205,24 @@ public synchronized void stopReporting()
         locks.forAll( ( jf ) -> {
             final StringBuilder owner = new StringBuilder();
 
-            final LockOwner ref = jf.getLockOwner();
-
-            if ( ref == null )
+            if ( jf.isWriteLocked() )
             {
-                owner.append( "UNKNOWN OWNER; REF IS NULL." );
+                owner.append( " (JoinableFile locked as WRITE)" );
```

Comment:
Seems this "owner" StringBuffer is never used for reporting. We appended many contents in it, but never trigger the reading of its content by toString() in the following code.